### PR TITLE
Update remote source path for cachito

### PR DESCRIPTION
**Pull request by leifmadsen**

### CODE REVIEW

**Comment by leifmadsen** [comment]
Adjustments based on documentation at https://osbs.readthedocs.io/en/latest/users.html#remote-sources

File: release/ansible/Dockerfile.rhel8 [hunk 1]
```diff
@@ -22,8 +22,11 @@ ENV HOME=/opt/ansible \
     USER_UID=1001
 
 # get artifacts from cachito
-COPY $REMOTE_SOURCES/cachito-gomod-with-deps/app $REMOTE_SOURCES_DIR
-WORKDIR $REMOTE_SOURCES_DIR/app
+COPY $REMOTE_SOURCES $REMOTE_SOURCES_DIR
+
+# TODO: remove this line (debugging)
```
Comment (oceanc80) [inline release/ansible/Dockerfile.rhel8:27]: Did we still want this debugging line?

Comment (everettraven) [reply release/ansible/Dockerfile.rhel8:27]: Just got out of the ansible call. I think we want this for now just to verify things are working as expected

**Review by everettraven — APPROVED** [review]
/lgtm

File: release/ansible/Dockerfile.rhel8 (hunk 1 above)
Comment (leifmadsen) [reply release/ansible/Dockerfile.rhel8:27]: Yea, I want it for this import so I can see what is going on. I'll remove it in another PR once we know things are working.

**Pull request merged**